### DISCUSSION TITLE
Problem using template field 'slack_msg_color'

### DISCUSSION THREAD

**Discussion by sl-maximevendrand** [cloud-custodian/cloud-custodian#7379] [General]
Hello, I'm trying to change the color of the slack messages using the template field 'slack_msg_color'. But it doesn't change the default value wich is 'danger'. 

This is my code for the notify action :
actions:
      - type: notify
        slack_template: slack_default
        slack_msg_color: good
        violation_desc: ""
        action_desc: ""
        to:
          - slack webhook
        transport:
          type: sqs
          queue: 

I am using the latest version of custodian 0.9.4
Is it possible that there is an error in the template slack_default ?

**Comment by kapilt** [comment]
the template is shipped with c7n_mailer package, and the slack color change option isn't available till the next release.

**Comment by sl-maximevendrand** [comment]
Ok thank you for the answer
